R/get_s.R

```diff
@@ -0,0 +1,13 @@
+#' @export
+get_s <- function(mt){
```

Comment:
I don't fully understand why the data frame created with `hail_dataframe()` does not contain the `s` identifier, so worth renaming to `hail_id()`. That said, I think in an ideal situation this gets extracted from `df` not the `mt`, but I might be missing something here. Do you have a better sense of why we need to extract the ideas from `mt`?

Reply:
In `matrixtable` we have entries, rows and cols. `s` is a col and have the same length than `entries`, but I'm not sure if it is in the same order to use `bind_cols`, for example.